### ISSUE TITLE
Javascripte bündeln

### ISSUE THREAD

**Issue by svenkaemper** [wisydb/wisy#22]
Analog zu den Stylesheets  #(18) sollte man versuchen, die Anzahl der Javascriptaufrufe zu reduzieren und nach Möglichkeit alle erforderlichen Skripte in einem externen, minifizierten Skript unterzubringen.

**Comment by wisydb** [comment]
Ja. Die Bündelung ist auf der Todo-Liste (Juni/Juli) mit Abarbeitung der SEO-Punkte. Detail-Absprache von J.M. und D. Bagel nötig. Auch bei der neuen Suche, ist dieser Punkt im Quellcode seitens D. Bagel vermerkt / vorgesehen...
